src/docs/product/accounts/membership.mdx

```diff
@@ -9,87 +9,128 @@ description: "Learn about the different types of Sentry membership, roles, and a
 
-Open membership is enabled by default, which allows members to join, leave, and add other members to teams.
+Only Org Admins, Managers, and Owners can add new teams to an organization. Users can join a team with one of the following team-level roles:
 
-## Restricting Access
+- **Team Contributors** can view and act on issues and, depending on org rules, add members and projects to the team.
+- **Team Admins** have additional permissions to manage their team's membership and projects. These permissions are granted at the team-level, and do not apply to teams where they are not a Team Admin.
 
-- Access to a Project is limited to the Team that owns the project. However, with open membership,
-  members can join and add members to teams.
-- To further restrict access to a Project, you can control access to Teams by making them accessible
-  only through invitation by a Manager, Owner, or Admin.
+Org Members can be Team Admins for specific teams, but they cannot add new teams. Org Admins automatically become Team Admins when they join a team. Org Managers and Owners can perform Team Admin actions for all teams without needing to join.
 
-To restrict Team access, go to Organization Settings and flip the “Open Membership” toggle.
+### Permissions
+
+Team-level roles grant more granular permissions to a specific team, as described in the table below:
+
+| **Action**                                                                                              | **Team Contributor** | **Team Admin**                                                         |
+| ------------------------------------------------------------------------------------------------------- | -------------------- | ---------------------------------------------------------------------- |
+| Can view and act on issues (such as assigning and resolving)                                            | ✓                    | ✓                                                                      |
+| Can join and leave teams (depending on open membership settings)                                        | ✓                    | ✓                                                                      |
+| Can add repositories                                                                                    | ✓                    | ✓                                                                      |
+| Can invite new members to join (_Invite will land in pending state for Managers and Owners to approve_) | ✓                    | ✓                                                                      |
+| Can change project settings                                                                             |                      | ✓ (_Only for projects they have access to_)                            |
```

Comment:
Seems a bit repetitive. Can we try something like this

```suggestion
| Can change project settings                                                                             |                      | ✓ **                            |
```

Then right below the table
```
[**] Only for teams or projects that they are an existing admin for.

Reply:
Good suggestion, updated!